asyncpg/types.py

```diff
@@ -85,6 +85,52 @@ def upper_inf(self):
     def isempty(self):
         return self._empty
 
+    def issubset(self, other):
+        if self._empty:
+            return True
+        if other._empty:
+            return False
+
+        if other._lower is None:
+            lowerOk = True
+        else:
+            if self._lower is None:
+                return False
```

Comment:
It would read better if you set `lowerOk = False` here and use an `elif`.  Also, please use `lower_ok` for the variable name for style consistency.